<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>351</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>定时器</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>开始</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnEnd">
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="btnOneShot">
      <property name="text">
       <string>动态创建单次定时器</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>定时周期</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBoxIntv"/>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="radioCoutinue">
      <property name="text">
       <string>连续定时</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioOneShot">
      <property name="text">
       <string>单次定时</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>123</y>
     <width>353</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>定时器精度</string>
   </property>
   <widget class="QRadioButton" name="radioPrecse">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>97</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>PreciseTimer</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioCoarse">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>95</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CoarseTimer</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioVeryCoarse">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>VeryCoarseTimer</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>353</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>当前时间(小时：分：秒)</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="lcdHour">
       <property name="digitCount">
        <number>2</number>
       </property>
       <property name="value" stdset="0">
        <double>23.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdMinute">
       <property name="digitCount">
        <number>2</number>
       </property>
       <property name="value" stdset="0">
        <double>23.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdSecond">
       <property name="digitCount">
        <number>2</number>
       </property>
       <property name="value" stdset="0">
        <double>23.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>流逝的时间：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
